src/bin/nodes/CommandFind.ts

```diff
@@ -87,7 +87,7 @@ class CommandFind extends CommandPolykey {
         );
         // Like ping it should error when failing to find node for automation reasons.
         if (!result.success)
-          throw new CLIErrors.ErrorNodeFindFailed(result.message);
+          throw new CLIErrors.ErrorCLINodeFindFailed(result.message);
```

Comment:
If you cannot resolve a node id on the DHT, doesn't it just loop forever? Therefore by default, the command can just run forever looking for the node until it appears. No need to throw an error here.

Furthermore if there is a timeout applied, which should be an option, then this is domain error of `nodes`, not `CLI`.

Reply:
I think having it go forever and having it timeout both have utility. I'll add an option for a timeout.

Reply:
Yea it's just default behaviour it can just run forever. Then optionally the user can add a timeout. These options should be put into `bin/utils/options.ts`.

Reply:
It now defaults to trying forever. I added two options to specify a timeout, you can specify how many times to attempt and the delay between attempts.

```ts
const attemptsCount = new commander.Option(
  '-c --count <count>',
  'Number of attempts before failing'
)
  .argParser(binParsers.parseNumber);

const attemptsDelay = new commander.Option(
  '-ad --attempt-delay <delay>',
  'Number of milliseconds between each attempt'
)
  .argParser(binParsers.parseNumber)
  .default(5000);
  ```

Reply:
Cool but try to use the same names we've used elsewhere. `--retry X` and `--retry-interval` or `--retry-timeout`.